### PR TITLE
#122 ref: create style for button and refactor start_activity

### LINKED ISSUE #122: [FRONT] - Refactoring: refactor page "old_start_activity"
change style page "old_start_activity" and change page "new_start_activity"

### PR DESCRIPTION
**Pull request by m83city**
add new image  refactoring start_activity, create nes style

### CODE REVIEW

File: app/src/main/res/drawable/button_new.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/drawable/button_new.xml]: rename this one to button_backgroung, if such element is exist rename that to old_button_background

File: app/src/main/res/layout/start_activity_new.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:layout_below="@id/logo_local"
+        android:orientation="vertical"
+        android:weightSum="5">
+
+        <Button
+            android:id="@+id/Sign_In_activity_fir"
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: change this to sign_in_id

File: app/src/main/res/layout/start_activity_new.xml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            style="@style/button_style_new"
+            android:text="@string/Sign_In" />
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: we have such text with multilanguage in values/button_text, if not exist create your own

File: app/src/main/res/layout/start_activity_new.xml [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+        <androidx.legacy.widget.Space
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1" />
+
+        <Button
+            android:id="@+id/Sign_Up_activity_fir"
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: -> create_account_id

File: app/src/main/res/layout/start_activity_new.xml [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+            style="@style/button_style_new"
+            android:text="@string/Sign_Up" />
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: in button_text we have button_create_new_account

File: app/src/main/res/layout/start_activity_new.xml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="20dp"
+            android:layout_marginEnd="20dp"
+            android:layout_weight="1"
+            android:orientation="horizontal"
+            android:weightSum="3">
+
+            <ImageButton
+                style="@style/button_style_new"
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: change old style to button_style_old, and this one withot suffix

File: app/src/main/res/layout/start_activity_new.xml [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+                android:layout_marginStart="0dp"
+                android:layout_weight="1"
+                android:contentDescription="@string/descr_google"
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: for description we have resources in content_description.xml

File: app/src/main/res/layout/start_activity_new.xml [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+                android:src="@drawable/google_new" />
+
+            <androidx.legacy.widget.Space
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: what is the widget?

File: app/src/main/res/layout/start_activity_new.xml [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_weight="1" />
+
+            <ImageButton
+                style="@style/button_style_new"
+                android:layout_marginEnd="0dp"
+                android:layout_weight="1"
+                android:contentDescription="@string/descr_twitter"
```
Comment (vlad-vinskevitch) [inline app/src/main/res/layout/start_activity_new.xml]: the same

File: app/src/main/res/values/strings.xml [hunk 10]
```diff
@@ -284,6 +284,8 @@
     <string name="descr_auto_tuning">Auto tuning</string>
     <string name="descr_add_meal">Add meal</string>
     <string name="descr_change_weight">Change weight</string>
+    <string name="descr_google">Sign in with google</string>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values/strings.xml]: remove this two and use content_description.xml for such resources

File: app/src/main/res/values/strings.xml [hunk 11]
```diff
@@ -284,6 +284,8 @@
...
+    <string name="descr_twitter">Sign in with twitter</string>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values/strings.xml]: the same

File: app/src/main/res/values/strings.xml [hunk 12]
```diff
@@ -387,7 +389,7 @@
     <string name="toast_field_value_to_long">Field value too long : %s</string>
     <string name="toast_field_value_to_short">Field value too short : %s</string>
     <string name="toast_field_must_be_not_empty">Field must be not empty : %s</string>
-    <string name="title_activity_main_meny">MainMeny</string>
+    <string name="title_activity_main_meny">Main Menu</string>
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values/strings.xml:391]: remove this and everything what related with lables or titels should be in text_view_field folder, never don't use string.xml

File: app/src/main/res/values/styles.xml [hunk 13]
```diff
@@ -1,6 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
+    <style name="button_style_new" parent="Theme.AppCompat.DayNight">
```
Comment (vlad-vinskevitch) [inline app/src/main/res/values/styles.xml]: rename this one without suffix, and add suffix to old

**Review by vlad-vinskevitch — CHANGES_REQUESTED** [review]

File: app/src/main/res/drawable/button_new.xml (hunk 1 above)
Comment (m83city) [reply app/src/main/res/drawable/button_new.xml]: i'm rename on button_backgroung_new and all files which do'n have sufix new i'm delete on #127 ticket

Comment (vlad-vinskevitch) [reply app/src/main/res/drawable/button_new.xml]: okay

File: app/src/main/res/layout/start_activity_new.xml (hunk 6 above)
Comment (m83city) [reply app/src/main/res/layout/start_activity_new.xml]: i can't change old style button becouse it affected all button project

File: app/src/main/res/layout/start_activity_new.xml (hunk 8 above)
Comment (m83city) [reply app/src/main/res/layout/start_activity_new.xml]: "Space" he need for correct placing in Linear Layout it is just spacing

**Review by vlad-vinskevitch — APPROVED** [review]

**Pull request merged**